### PR TITLE
Issue #SB-3647 fix: Add function test for badge issuer creation

### PR DESCRIPTION
**Pull request by bvinayakumar**

### CODE REVIEW

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java [hunk 1]
```diff
@@ -51,22 +60,34 @@ public static boolean doDeleteOperation (String url) {
...
+	 * @param url HTTP URL to use in the request
+	 * @param formDataFile File path containing each form parameter in a new line in format (key=value)
+	 */
+	public void multipartPost(HttpClientActionBuilder httpClientActionBuilder, TestGlobalProperty config, String url, String formDataFile) {
+		MultiValueMap<String, Object> formData = new LinkedMultiValueMap<>();
+
+		try (Scanner scanner = new Scanner(new File(getClass().getClassLoader().getResource(formDataFile).getFile()))) {
+
+			while (scanner.hasNext()) {
+				String[] param = scanner.nextLine().split("=");
```
Comment (manzarul) [inline sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: we can use constant for equal delimiter  for "="

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java [hunk 2]
```diff
@@ -51,22 +60,34 @@ public static boolean doDeleteOperation (String url) {
...
+				if (param.length == 2) {
```
Comment (manzarul) [inline sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: is it possible in some case param can be null.

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java [hunk 3]
```diff
@@ -51,22 +60,34 @@ public static boolean doDeleteOperation (String url) {
...
 			}
-		} catch (Exception e) {
+
+		} catch (IOException e) {
 			e.printStackTrace();
 		}
-		return result.toString();
+
+		httpClientActionBuilder.send().post(url)
+				.contentType("multipart/form-data")
```
Comment (manzarul) [inline sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: we can create content type enum in Constant "multipart/form-data"

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java (hunk 1 above)
Comment (bvinayakumar) [reply sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: done

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java (hunk 2 above)
Comment (bvinayakumar) [reply sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: done

File: sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java (hunk 3 above)
Comment (bvinayakumar) [reply sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java]: Used MediaType.MULTIPART_FORM_DATA.

**Pull request merged**